### PR TITLE
Update app-wrapper-prepare-ios.md for -ac and -ar to be required params

### PR DESCRIPTION
**Pull request by tranamst**
Update app-wrapper-prepare-ios.md for -ac and -ar to be required params

### CODE REVIEW

File: memdocs/intune/developer/app-wrapper-prepare-ios.md [hunk 1]
```diff
@@ -233,9 +233,9 @@ You can use the following command line parameters with the App Wrapping Tool:
 |**-p**|`<Path of your provisioning profile for iOS apps>`|
 |**-c**|`<SHA1 hash of the signing certificate>`|
 |**-h**| Shows detailed usage information about the available command line properties for the App Wrapping Tool. |
+|**-ac**|`<Client ID of the input app if the app uses the Microsoft Authentication Library>` This is the guid in the Client ID field from your app's listing in the App Registration blade. |
```
Comment (JohanFreelancer9) [inline memdocs/intune/developer/app-wrapper-prepare-ios.md:236]:
```suggestion
|**-ac**|`<Client ID of the input app if the app uses the Microsoft Authentication Library>` This is the GUID in the Client ID field from your app's listing in the App Registration blade. |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tranamst, kindly commit the suggested change. Many thanks.

**Comment by tranamst** [comment]
> Copy review - @tranamst, kindly commit the suggested change. Many thanks.

Thanks for reviewing. I have updated PR with the suggested change. Could you merge this in? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]
@tranamst, I only do a copy edit. The author or manager will merge it once the changes are approved. Thanks!

**Comment by tranamst** [comment]
@Erikre Bumping this - Could you take a look and approve? Thank you!

**Comment by tranamst** [comment]
@JohanFreelancer9 Could you help me ping someone on the reviewer team to approve this for me? Thank you.

**Comment by JohanFreelancer9** [comment]
> @JohanFreelancer9 Could you help me ping someone on the reviewer team to approve this for me? Thank you.

@yogkumgit, could you possibly assist? Many thanks.

**Comment by yogkumgit** [comment]
@Erikre Please review and sign off this PR. Thanks

**Comment by tranamst** [comment]
@yogkumgit Can you review and sign this PR off? It has been 3 weeks.

**Comment by yogkumgit** [comment]
@Erikre Gentle reminder. Thanks

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Pull request merged**